src/components/divider/divider.scss

```diff
@@ -0,0 +1,15 @@
+@import 'variables';
+@import 'default-theme';
+
+md-divider {
```

Comment:
Not sure about that PR in general, but the divider is already implemented in the list component (as it belongs to that)
- See https://github.com/angular/material2/blob/master/src/components/list/list.scss#L156